### PR TITLE
added loadmore feature

### PR DESCRIPTION
**Pull request by RaymondLi-1993**
added loadmore feature so the user is not overwhelmed with drinks

### CODE REVIEW

File: app/javascript/components/pages/Home.js [hunk 1]
```diff
@@ -18,6 +31,11 @@ class Home extends Component {
                         <CardTitle tag="h5">
                           Cocktail {drink.drinkName}
                         </CardTitle>
+                        <img 
+                        style={{width:"100px", height:"100px"}}
```
Comment (minachuong) [inline app/javascript/components/pages/Home.js:35]: there's a mix of custom css in a file like `"home-cards"` what i think suppose to be coming from an imported file and inline styling. it'd be best practice to stick to one place for where the styles live

File: app/javascript/components/pages/Home.js [hunk 2]
```diff
@@ -2,14 +2,27 @@ import React, { Component } from "react";
...
 class Home extends Component {
+  constructor(props){
+    super(props);
+    this.state = {
+      count:3,
+    }
+  }
+
+  loadMore = () => {
+    this.setState(prev => ({ count: prev.count + 3 }));
```
Comment (minachuong) [inline app/javascript/components/pages/Home.js:13]: nice use of the updater callback

File: app/javascript/components/pages/Home.js [hunk 3]
```diff
@@ -2,14 +2,27 @@ import React, { Component } from "react";
...
+  }
+
+
   render() {
     let { drinks } = this.props;
+    console.log(this.state.count)
```
Comment (minachuong) [inline app/javascript/components/pages/Home.js:19]: 👀 keep the console clean, please remove

**Review by minachuong — APPROVED** [review]
👍

**Comment by RaymondLi-1993** [comment]
ok got it will fix that inline style on the next fix. I just wanted to use it to see how the display will work.

**Comment by RaymondLi-1993** [comment]
Got it! Ty! I will change out of the inline style on the next push I was just using it as a test to see the images.

**Pull request merged**